.github/workflows/actions.yml

```diff
@@ -250,7 +250,16 @@ jobs:
           xcodebuild -workspace codebase/temp/solutions/HelloXCode_mac.xcworkspace -configuration ${{ matrix.configuration }} -scheme exe_mac
           cd codebase/temp/bin/mac_${{ matrix.configuration }}
           DYLD_LIBRARY_PATH=. ./exe
-
+          
+      - name: HelloAndroid ${{ matrix.configuration }}
+        if: runner.os == 'Windows'
+        working-directory: 'samples/HelloAndroid'
+        run: |
+          & $env:SHARPMAKE_EXE "/sources('HelloAndroid.main.sharpmake.cs')"
+          & $env:ANDROID_HOME/tools/bin/sdkmanager --uninstall 'build-tools;31.0.0'
```

Comment:
why do you uninstall? please add a comment explaining it, and the version number maybe could be set as a variable of the job and used here to ease maintenance